src/main/webapp/about.html

```diff
@@ -5,11 +5,11 @@
       <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no, minimal-ui">
       <title>about</title>
       <!-- css -->
-      <link type="text/css" rel="stylesheet" href="./assets/css/mainStyle.css">
```

Comment:
Why is this removed?

Reply:
For this PR, I accidentally included commits that weren't merged to master yet so I reverted them here.

Reply:
Commits from a different PR that hadn't been merged yet? :0